docarray/base_doc/mixins/io.py

```diff
@@ -146,7 +146,7 @@ def __bytes__(self) -> bytes:
         return self.to_bytes()
 
     def to_bytes(
-        self, protocol: str = 'protobuf', compress: Optional[str] = None
+        self, protocol: Literal['pickle', 'protobuf'], compress: Optional[str] = None
```

Comment:
we need to keep the default vlaue

Reply:
Updated the changes on all the occurrences.